core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -121,11 +121,22 @@ var SkillEditorPage = function() {
   var conceptCardExplanationEditorInput = element(
     by.css('.protractor-test-concept-card-text .protractor-test-rte'));
 
+  var staleTabInfoModal = element(
+    by.css('.protractor-test-stale-tab-info-modal'));
+  var unsavedChangesStatusInfoModal = element(
+    by.css('.protractor-test-unsaved-changes-info-modal'));
+
   this.get = async function(skillId) {
     await browser.get(EDITOR_URL_PREFIX + skillId);
     await waitFor.pageToFullyLoad();
   };
 
+  this.openSkillEditorInNewTab = async function(skillId) {
+    await browser.executeScript(
+      'window.open("' + EDITOR_URL_PREFIX + skillId + '")');
```

Comment:
Can you open the skill editor by navigating through the UI like a user would instead?

Reply:
Done.